django_future_tasks/fields.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+from cronfield.models import CronField
+from django.core.exceptions import ValidationError
+
+
+def _validate_CRON_string(value):
+    """Validation routine for CRON string in TestingPlan"""
+
+    if value.strip() != value:
+        raise ValidationError("Leading nor trailing spaces are allowed")
+    columns = value.split()
+    if columns != value.split(" "):
+        raise ValidationError("Use only a single space as a column separator")
+
+    if len(columns) not in [5, 6]:
+        raise ValidationError("Entry has to consist of 5 or 6 columns")
+
+    pattern = r"^(\*|\d+(-\d+)?(,\d+(-\d+)?)*)(/\d+)?$"
+    p = re.compile(pattern)
+    for i, c in enumerate(columns):
+        if not p.match(c):
+            raise ValidationError("Incorrect value {} in column {}".format(c, i + 1))
+
+
+class FuturetaskCronField(CronField):
```

Comment:
For the sake of consistency I would suggest
```suggestion
class FutureTaskCronField(CronField):
```

Reply:
Fixed